<?xml version="1.0" encoding="UTF-8"?>
<launch>
   <node pkg="assignment_1" type="control" name="setpose_pub" output="screen"/>
   <node pkg="assignment_1" type="control" name="feedback_controller" output="screen"/>
   <rosparam file="$(find assignment_1)/config/params.yaml" />

</launch>
